wire-gson-support/src/test/java/com/squareup/dinosaurs/Dinosaur.java

```diff
@@ -0,0 +1,252 @@
+// Code generated by Wire protocol buffer compiler, do not edit.
```

Comment:
did you actually copy paste those?

Reply:
Yes I did! We could generate them instead but it's less urgent.